### PR TITLE
chore: parse changelog PR author names from contributors

### PR DESCRIPTION
**Pull request by theseion**

### CODE REVIEW

**Comment by dune73** [comment]
Haha, open review that I did not finish again. Sorry.

File: .github/create-changelog-prs.py [hunk 1]
```diff
@@ -108,7 +111,7 @@ def generate_content(prs: list, merged_by: str) -> (str, str):
 	for pr in prs:
 		pr_number = pr["number"]
 		pr_title = pr["title"]
-		pr_author = pr["author"]["login"]
+		pr_author = get_pr_author_name(pr["author"]["login"])
 		new_line = f"* {pr_title} (@{pr_author}) [#{pr_number}]\n"
```
[the remark this answers is not in the document]
Comment (theseion) [reply .github/create-changelog-prs.py:115]: Because not all of us use their real names on GH.

[the remark this answers is not in the document]
Comment (dune73) [reply .github/create-changelog-prs.py:115]: Ah, it's slowly dawning on me. And in the contributor file, we have a comprehensive mapping table.

**Review by dune73 — APPROVED** [review]

**Comment by dune73** [comment]
Thanks for the update. Merging now.

**Pull request merged**